### PR TITLE
Add privacy configuration option.

### PR DESCRIPTION
**Pull request by Informatic**
`IrisCore` and `raven` initialization has been moved to `Frontend` class to have proper access to configuration.

`/iris/http/config.js` endpoint returning generated JS code that sets `window._iris_config` to contents of mopidy `[iris]` configuration section has been created. This was the simplest way to have early configuration options without making `/index.html` a template generated with a http handler. (to disable ReactGA) This could be expanded further, to actually use these values instead of calling `get_config`, when available.

Separate config option to specify allowed CORS hosts would be quite beneficial, but right now let's just assume user doesn't need remote control when using privacy mode.

Privacy by default.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -33,7 +33,7 @@
 	</div>
 
 </body>
-
+<script src="/iris/http/config.js"></script>
```
Comment (jaedb) [inline src/index.html]: Instead of including code at render, this should be handled _within_ the app itself (ie at event `PUSHER_CONNECTED`)

**Comment by haizop** [comment]
So, is this branch/feature dead in the water? I would really like to use Iris, but google analytics are a total non starter for me, even anonymized. Thanks to @jaed for all the great work you have put into this project. I really hope you support full disabling of google analytics.

**Comment by Informatic** [comment]
@haizop currently `feature/privacy` branch in my fork (ie. one this pull request is referencing) is directly rebased (and tested) over current `master` in main project. I really hope to find some time to work on this further (as @jaedb suggested) and pull it into upstream.

**Comment by Informatic** [comment]
Keep in mind I also didn't manage to get rid of Google Analytics completely, `.js` is still loaded from Google, but it is not initialized in Iris.

**Comment by jaedb** [comment]
If you're concerned about Analytics then you'll likely need to block Analytics from almost every website you visit. The practical approach to this would be to install a browser extension to block all calls to Google Analytics.

That being said, this is still on the to-do list. There are some more pressing functional issues to address before I can get to this.

**Comment by haizop** [comment]
Thanks @jaedb and @Informatic for the updates.

**Comment by jaedb** [comment]
Instead of incorporating privacy settings server-wide, it is now client-specific. This means users themselves can configure their preferred privacy level.

This enables control as a simple checkbox in the UI. The introduction of an initial setup for new users will make this more transparent and easily accessible. It will be editable also under _Settings > Privacy_ at any time.

**Pull request closed without merging**